lua/terrortown/entities/roles/jester/shared.lua

```diff
@@ -254,7 +254,7 @@ if SERVER then
 			end
 		end
 		
-		if ply:GetSubRole() == ROLE_JESTER and not (IsValid(attacker) and attacker:IsPlayer() and INFECTED and attacker:GetSubRole() == ROLE_INFECTED) then
+		if ply:GetSubRole() == ROLE_JESTER and (not IsValid(attacker) or not attacker:IsPlayer() or attacker ~= ply and (INFECTED and attacker:GetSubRole() ~= ROLE_INFECTED)) then
```

Comment:
This change is wrong too. It shall just happen if you are a Jester and you die, except if you are killed by an Infected (because you get converted later). Now it's done all the time except if you get killed by an infected, but additionally, if there wasn't a killer and the winstates doesn't trigger.

Reply:
nvm, i did the mistake month ago, lol